USAGE.rst

```diff
@@ -93,19 +95,21 @@ Java (pubsub)
   ::
 
      python execute_pipeline.py \
-        --config "../googleapis/gapic/core/artman_core.yaml:core,\
-        ../googleapis/gapic/lang/common.yaml:default|java" \
+        --config "../googleapis/gapic/core/artman_core.yaml,\
+        ../googleapis/gapic/lang/common.yaml" \
         JavaCorePipeline
```

Comment:
It does have this, CoreProtoPipeline, I just missed it here in the usage. Fixed now.